### ISSUE TITLE
Linux freezing on latest yuzu flatpak release

### ISSUE THREAD

**Issue by omanwhatiscomputer** [flathub/org.yuzu_emu.yuzu#232]
The latest flatpak release freezes Pop OS 21.04 (Gnome 3.38.5) system on yuzu launch. No error logs, no error messages, nothing. (forces manual system restart)

Distro: Pop OS  21.04 (Gnome 3.38.5)
API: X11

Version: mainline-0-834
System Specs:
[Vulkan] AMD strix rx570 4GB
Ryzen r5 3600; 16 GB ddr4 ram

More detailed info:
```
System:
  Kernel: 5.15.5-76051505-generic x86_64 bits: 64 
  compiler: gcc v: 10.2.1 Desktop: GNOME 3.38.4 Distro: Pop!_OS 21.04 
  base: Ubuntu 21.04 Hirsute 
Machine:
  Type: Desktop System: Gigabyte product: A320M-S2H v: N/A 
  Mobo: Gigabyte model: A320M-S2H-CF
  UEFI: American Megatrends LLC. v: F54c date: 09/30/2021 
CPU:
  Info: 6-Core model: AMD Ryzen 5 3600 bits: 64 type: MT MCP arch: Zen 2 
  rev: 0 L2 cache: 3 MiB 
Graphics:
  Device-1: AMD Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] 
  vendor: ASUSTeK driver: amdgpu v: kernel bus ID: 07:00.0 
  Display: x11 server: X.Org 1.20.11 driver: loaded: amdgpu,ati 
  unloaded: fbdev,modesetting,vesa resolution: 1920x1080~60Hz 
  OpenGL: renderer: Radeon RX 570 Series (POLARIS10 DRM 3.42.0 
  5.15.5-76051505-generic LLVM 12.0.0) 
  v: 4.6 Mesa 21.2.2 direct render: Yes 
```

**Comment by liushuyu** [comment]
This is very strange. I suspect this to be a kernel or graphics driver problem.

You could check kernel logs from the last boot using `sudo journalctl -b -1 -k` and see if there are warnings or errors there.

**Comment by liushuyu** [comment]
Probably related: https://github.com/yuzu-emu/yuzu/issues/7463

**Comment by omanwhatiscomputer** [comment]
UPDATE: The problem still persists after doing a fresh install (and deleting all the folders in "org.yuzu_emu.yuzu"). Everything seems to work until I attach the game folder with all the .nsp files. The whole system freezes after that.

**Comment by liushuyu** [comment]
> Everything seems to work until I attach the game folder with all the .nsp files.

Oh, so you didn't boot any games? The system froze right after you select the game folder?

**Comment by omanwhatiscomputer** [comment]
Yes, that is correct.

**Comment by dysfunctionalriot** [comment]
Had a similar issue with the flatpak version, but it would only freeze on opening a game a second time, or restarting yuzu without restarting my pc. Logs were also completely blank after the freeze. Appimage version works properly. I have already mentioned the problem on discord. 
Pop_os 21.04
And ryzen 7 4000u
Integrated graphics
12 GB ram 16 GB swap
Multiple yuzu versions

**Comment by rycnot** [comment]
Same issue with flatpak version on openSUSE Tumbleweed. System freezes the moment yuzu launch. Appimage is  working fine.

[kernel_log.txt](https://github.com/flathub/org.yuzu_emu.yuzu/files/7672191/kernel_log.txt)

System Specs:
AMD FX-8320
RX 570 8GB
openSUSE Tumbleweed (kernel 5.15.5)

**Comment by liushuyu** [comment]
Thank you all for reporting. I am currently investigating this issue.
Can @rycnot @dysfunctionalriot confirm which desktop environment are you using? GNOME perhaps?

I think #182 is also probably related.

**Comment by rycnot** [comment]
I'm on latest KDE. I upgraded to kernel 5.15.6 and the issue was gone. Then booted back to 5.15.5  - system froze, probably some kernel bug that was.

**Comment by liushuyu** [comment]
> I'm on latest KDE. I upgraded to kernel 5.15.6 and the issue was gone. Then booted back to 5.15.5 - system froze, probably some kernel bug that was.

I see. Can other users in this thread confirm upgrading the system would solve the issue?

**Comment by liushuyu** [comment]
Ping @dysfunctionalriot @omanwhatiscomputer, do you still have this issue after upgrading your systems?

**Comment by ijaure299** [comment]
I can confirm it is the same issue for me using Pop OS 21.10 (Gnome)

**Comment by omanwhatiscomputer** [comment]
Yes, the issue still persists for me on Pop OS 21.10 and it is still using kernel version 5.15.5. I suppose it really is a kernel issue since this version is known for being very unstable for some users but I could be wrong

**Comment by liushuyu** [comment]
> I can confirm it is the same issue for me using Pop OS 21.10 (Gnome)


> Yes, the issue still persists for me on Pop OS 21.10 and it is still using kernel version 5.15.5. I suppose it really is a kernel issue since this version is known for being very unstable for some users but I could be wrong

If this is a kernel issue or a Pop OS issue, then there is not much I could do about it.

Have you tried to ask Pop OS developers for information on the kernel issues? I have yet to find other users using other distros that experience a similar issue.

**Comment by ijaure299** [comment]
> 

I have no problems what so ever with other apps, everything works 100% so I'm assuming its related to the flatpak version since I heard people reported that the appimage version doesn't have this problem

**Comment by omanwhatiscomputer** [comment]
> > I'm on latest KDE. I upgraded to kernel 5.15.6 and the issue was gone. Then booted back to 5.15.5 - system froze, probably some kernel bug that was.
> 
> I see. Can other users in this thread confirm upgrading the system would solve the issue?

The issue has been resolved after upgrading kernel to version 5.15.8. The flatpak version now works as intended.
